sru/custom_m_utils.py

```diff
@@ -0,0 +1,188 @@
+from torch import nn
+import logging
+from typing import List, Union
+from sru import SRU, SRUCell
+
+logger = logging.getLogger(__name__)
+
+
+def convert_sru_cell_to_linears(cell: SRUCell):
```

Comment:
Is this a public method that will be exposed to other libraries?
If so, mind if you add a doc string? also, these are inplace operations to the given `cell`.